esti/ops/docker-compose-dynamodb.yaml

```diff
@@ -1,71 +1,35 @@
 version: "3"
 services:
   lakefs:
-    image: "${REGISTRY:-treeverse}/${REPO:-lakefs}:${TAG:-dev}"
-    command: "${COMMAND:-run}"
-    ports:
-      - "8000:8000"
+    extends:
+      file: esti/ops/docker-compose-common.yaml
+      service: lakefs
     depends_on:
       - "dynamodb"
     volumes:
       - lakefs-app:/app:ro
     environment:
-      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY="some random secret string"
-      - LAKEFS_AUTH_UI_CONFIG_RBAC=${LAKEFS_AUTH_UI_CONFIG_RBAC:-simplified}
-      - LAKEFS_AUTH_API_ENDPOINT=${LAKEFS_AUTH_API_ENDPOINT:-http://localhost:8001/api/v1}
-      - LAKEFS_BLOCKSTORE_TYPE=${LAKEFS_BLOCKSTORE_TYPE:-local}
-      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
-      - LAKEFS_BLOCKSTORE_LOCAL_IMPORT_ENABLED=true
-      - LAKEFS_BLOCKSTORE_LOCAL_ALLOWED_EXTERNAL_PREFIXES=/tmp
-      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
-      - LAKEFS_LOGGING_LEVEL=DEBUG
-      - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
-      - LAKEFS_STATS_ENABLED
-      - LAKEFSACTION_VAR=this_is_actions_var
       - LAKEFS_DATABASE_TYPE=dynamodb
       - LAKEFS_DATABASE_DYNAMODB_ENDPOINT=http://dynamodb:8000
       - LAKEFS_DATABASE_DYNAMODB_AWS_REGION=us-east-1
       - LAKEFS_DATABASE_DYNAMODB_AWS_ACCESS_KEY_ID=AKIAIO5FODNN7EXAMPLE
       - LAKEFS_DATABASE_DYNAMODB_AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K3MDENG/bPxRfiCYEXAMPLEKEY
-      - LAKEFS_USAGE_REPORT_ENABLED=false
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
 
-  dynamodb:    
-    image: "amazon/dynamodb-local:2.5.2"
-    ports:
-      - "6432:8000"
+  dynamodb:
+    extends:
+      file: esti/ops/docker-compose-common.yaml
+      service: dynamodb
 
   esti:
-    image: "golang:1.23-alpine"
-    links:
-      - lakefs:s3.local.lakefs.io
-      - lakefs:testmultipartupload.s3.local.lakefs.io
-      - lakefs:testmultipartuploadabort.s3.local.lakefs.io
-      - lakefs:testdeleteobjects.s3.local.lakefs.io
-      - lakefs:testmigrate-testpremigratemultipart.s3.local.lakefs.io
-      - lakefs:migrate.s3.local.lakefs.io
+    extends:
+      file: esti/ops/docker-compose-common.yaml
+      service: esti
     environment:
-      - CGO_ENABLED=0
-      - AWS_ACCESS_KEY_ID
-      - AWS_SECRET_ACCESS_KEY
-      - AWS_REGION=us-east-1
-      - ESTI_STORAGE_NAMESPACE
-      - ESTI_BLOCKSTORE_TYPE
-      - ESTI_AWS_ACCESS_KEY_ID
-      - ESTI_SETUP_LAKEFS
-      - ESTI_AWS_SECRET_ACCESS_KEY
-      - ESTI_ENDPOINT_URL=http://lakefs:8000
-      - ESTI_BINARIES_DIR=/app
       - ESTI_DATABASE_KV_ENABLED
```

Comment:
You already know the value - set it and remove the definition from the esti yaml job itself